api/Entity.go

```diff
@@ -72,17 +73,6 @@ func (vrsConnection *VRSConnection) CreateEntity(info EntityInfo) error {
 	return nil
 }
 
-// DestroyEntity removes an entity from the Nuage VRS
```

Comment:
Wonder if we should keep this? Removing this method seems like breaking potential external integrations